duckduckgo/iOS
# Bump autoconsent to 5.1.0

Task/Issue URL: https://app.asana.com/0/0/1205171303632341/f
Tech Design URL:
CC:

**Description**:

Updating autoconsent. It contains a few fixes addressing breakage bugs.



**Steps to test this PR**:
1. Use this config: https://jsonblob.com/1135512371392798720
2. Verify that the pop-up is dismissed correctly on
  - https://www.buhl.de/mein-buhlkonto/ 
  - https://www.pcworld.com/



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
